### DISCUSSION TITLE
Replace in blocks in scripting editor

### DISCUSSION THREAD

**Discussion by Rinkton** [godotengine/godot-proposals#4290] [Scripting]
We can replace in whole script via `Ctrl+R` shortcut, we can replace in all files via `Ctrl+Shift+F` and "Replace" button. But we can't replace in script's block(in function or statement). Why don't add this opportunity?

It'll allow us replace some variable names in big scripts files and don't feel fear that changing variable in this small function don't cause some changes in other part of script we don't expect.
Also, if we create some function by copying another function and we need to change some variable names we can't just choose this copied function and replace in block some names instead of manually change variable names.
Also, this feature exists in other editors/IDE(e.g. Visual Studio).

It's can be realized by some new shortcut, like `Ctrl+R+B` and when we pressed this combination, depending in what function our cursor had placed, we can replace something in the block.
Also we can just highlight some block by mouse and then press `Ctrl+R`.

**Comment by Mickeon** [comment]
See also: https://github.com/godotengine/godot-proposals/issues/899

**Comment by Rinkton** [comment]
> Also we can just highlight some block by mouse and then press Ctrl+R.
...So, it's working like that in Godot. Maybe for a little bit harder, cuz we need to toggle "Selection Only" checkbox, but is working.
![image](https://user-images.githubusercontent.com/57531521/172055893-d26c1ed6-b19d-4e57-ab34-3998e2af018f.png)

(Sorry for silly issue, I just didn't noticed that)
